packages/web/src/features/printing-business/component/PrintingBusinessForm/PrintingBusinessFormThird.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from "react";
+import styled from "styled-components";
+import { setHours } from "date-fns";
+import Card from "@sparcs-clubs/web/common/components/Card";
+
+import TextInput from "@sparcs-clubs/web/common/components/Forms/TextInput";
+
+import { printingBusinessOrderSteps } from "@sparcs-clubs/web/constants/printingBusiness";
+import DesiredPickUpTimeSelection from "@sparcs-clubs/web/features/printing-business/component/DesiredPickUpTimeSelection/DesiredPickUpTimeSelection";
+
+import type { PrintingBusinessFormProps } from ".";
+import FormCheck from "./_atomic/FormCheck";
+
+interface PrintingBusinessFormThirdProps {
+  username: PrintingBusinessFormProps["username"];
+  clubs: PrintingBusinessFormProps["clubs"];
+  requestParam: PrintingBusinessFormProps["requestParam"];
+  requestForm: PrintingBusinessFormProps["requestForm"];
+  setRequestForm: PrintingBusinessFormProps["setRequestForm"];
+}
```

Comment:
```suggestion
type PrintingBusinessFormThirdProps = Pick<PrintingBusinessFormProps, 'username' | 'clubs' | 'requestParam' | 'requestForm' | 'setRequestForm'>;
```
💊 이렇게 하는건 어떨까요? 그런데 interface를 꼭 써야하면 이렇게 하면 안될거 같아요!

Reply:
좋습니다. 주요한 차이는 오버로딩이 가능한가 인것 같은데 그런 컴포넌트의 Props를 수정하여 사용할 일은 없으니 괜찮을 것 같아요!